shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/sctl/hint/ShardingCTLHintBackendHandler.java

```diff
@@ -43,7 +43,10 @@
     private final BackendConnection backendConnection;
     
     private HintCommandExecutor hintCommandExecutor;
-    
+
+    public ShardingCTLHintBackendHandler() {
```

Comment:
why add a constructor?

Reply:
Sorry,this was a mistake,Do I have a chance to resubmit it again?

Reply:
Yes, just git push